### PR TITLE
Campaigners mapping

### PR DESCRIPTION
**Pull request by jmuessel**
## Purpose of this PR
In the previous [PR ](https://github.com/pik-piam/piamInterfaces/pull/215) I introduced bugs that resulted from opening the csv with Excel. With this PR I am reversing this (e.g. piam_factor 1.10 --> 110., or deleting double quotes """"). 

Furthermore, I tried to use the w/o bunkers representation as often as possible to avoid double counting and to simplify the reporting.

### CODE REVIEW

File: inst/templates/mapping_template_CAMPAIGNers.csv [hunk 1]
```diff
@@ -242,8 +242,8 @@ NA;2;carbon management;Carbon Capture|Storage|Electricity;Mt CO2/yr;Carbon Manag
 210;2;emissions (CO2);Emissions|CO2|Energy|Demand|Industry|Pulp and Paper;Mt CO2/yr;;;;;;Fossil CO2 emissions from fuel combustion in industry in the pulp and paper sector;
 211;2;emissions (CO2);Emissions|CO2|Energy|Demand|Industry|Non-Ferrous Metals;Mt CO2/yr;;;;;;Fossil CO2 emissions from fuel combustion in industry in the non-ferrous metals sector;
 212;2;emissions (CO2);Emissions|CO2|Energy|Demand|Other Sector;Mt CO2/yr;;;;;;CO2 emissions from fuel combustion in other energy supply sectors (please provide a definition of other sources in this category in the 'comments' tab);
-213;2;emissions (CO2);Emissions|CO2|Energy|Demand|Residential;Mt CO2/yr;;;;not modelled but post-processed;;Fossil CO2 emissions from fuel combustion in residential (IPCC category 1A4b);
```
Comment (orichters) [inline inst/templates/mapping_template_CAMPAIGNers.csv:245]: do you want to delete this comment as well?

File: inst/templates/mapping_template_CAMPAIGNers.csv [hunk 2]
```diff
@@ -811,22 +811,22 @@ NA;2;carbon management;Carbon Capture|Storage|Electricity;Mt CO2/yr;Carbon Manag
 770;2;energy (final);Final Energy|Industry|Non-Ferrous Metals|Geothermal;EJ/yr;;;;;;Final energy consumption of geothermal heat by the non-ferrous metals sector;
 771;2;energy (final);Final Energy|Industry|Non-Ferrous Metals|Waste;EJ/yr;;;;;;Final energy consumption of non-renewable waste by the non-ferrous metals sector;
 772;2;energy (final);Final Energy|Industry|Non-Ferrous Metals|Other;EJ/yr;;;;;;Final energy consumption of other energy carriers by the non-ferrous metals sector (please provide a definition of the energy carriers in this category in the 'comments' tab);
-773;1;energy (final);Final Energy|Residential;EJ/yr;;;;not modelled but post-processed;;final energy consumed in the residential sector;
-774;2;energy (final);Final Energy|Residential|Electricity;EJ/yr;;;;not modelled but post-processed;;final energy consumption by the residential sector of electricity (including on-site solar PV), excluding transmission/distribution losses;
```
Comment (orichters) [inline inst/templates/mapping_template_CAMPAIGNers.csv:815]: same…

File: inst/templates/mapping_template_CAMPAIGNers.csv [hunk 3]
```diff
@@ -840,34 +840,35 @@ NA;2;carbon management;Carbon Capture|Storage|Electricity;Mt CO2/yr;Carbon Manag
...
+823;2;energy (final);Final Energy|Residential and Commercial|Liquids|Electricity;EJ/yr;;;;;;final energy consumption by the residential & commercial sector of liquid efuels ("power-to-liquid");
 824;2;energy (final);Final Energy|Residential and Commercial|Liquids|Fossil;EJ/yr;;;;;;final energy consumption by the residential & commercial sector of liquids from fossil origin (conventional & unconventional petroleum, coal-to-liquids, gas-to-liquids);
 825;1;energy (final);Final Energy|Transportation;EJ/yr;FE|Transport|w/o Bunkers;EJ/yr;;;;final energy consumed in the transportation sector, excluding bunker fuels, excluding pipelines;
-826;1;energy (final);Final Energy|Transportation (w/ bunkers);EJ/yr;FE|Transport edge;EJ/yr;;;;final energy consumed in the transportation sector, including bunker fuels, excluding pipelines;
+826;1;energy (final);Final Energy|Transportation (w/ bunkers);EJ/yr;FE|Transport|w/o Bunkers;EJ/yr;;;;final energy consumed in the transportation sector, including bunker fuels, excluding pipelines;
```
Comment (orichters) [inline inst/templates/mapping_template_CAMPAIGNers.csv:869]: do you want to add new variables here?

File: inst/templates/mapping_template_NAVIGATE.csv [hunk 4]
```diff
@@ -998,17 +998,23 @@ idx;Tier;Category;Variable;Unit;piam_variable;piam_unit;piam_factor;internal_com
 951;1;energy (final);Final Energy|Transportation|Bus|Hybrid|Electricity;EJ/yr;;;;;;final energy consumption of liquid fuels in the transportation sector by Hybrid Bus;
 952;1;energy (final);Final Energy|Transportation|Bus|Hybrid|Liquids;EJ/yr;;;;;;final energy consumption of liquid fuels in the transportation sector by Hybrid Bus;
 953;1;energy (final);Final Energy|Transportation|Bus|Hybrid|Gases;EJ/yr;;;;;;final energy consumption of gases in the transportation sector by Hybrid Bus;
+954;1;energy (final);Final Energy|Transportation|Passenger;EJ/yr;FE|Transport|Pass|Pass|w/o Bunkers;EJ/yr;;;;final energy consumed for passenger transportation;x
```
Comment (orichters) [inline inst/templates/mapping_template_NAVIGATE.csv:1001]: not sure what is going on here, these seem to be bigger changes…

**Review by orichters — APPROVED** [review]
the corrections look fine, there is some more stuff going (new variables etc.) which are not really covered by the PR description so I'm not sure whether those should be merged as well

File: inst/templates/mapping_template_NAVIGATE.csv (hunk 4 above)
Comment (jmuessel) [reply inst/templates/mapping_template_NAVIGATE.csv:1001]: Before there was a bug in the NAVIGATE template: Final Energy|Transportation|Passenger was mapped to FE|Transport|Liquids|Oil

I changed this to FE|Transport|Pass|Pass|w/o Bunkers

File: inst/templates/mapping_template_CAMPAIGNers.csv (hunk 3 above)
Comment (jmuessel) [reply inst/templates/mapping_template_CAMPAIGNers.csv:869]: This was a mistake - changed it.

File: inst/templates/mapping_template_CAMPAIGNers.csv (hunk 1 above)
Comment (jmuessel) [reply inst/templates/mapping_template_CAMPAIGNers.csv]: That was a mistake, thank you for pointing this out!

File: inst/templates/mapping_template_CAMPAIGNers.csv (hunk 2 above)
Comment (jmuessel) [reply inst/templates/mapping_template_CAMPAIGNers.csv]: That was a mistake, thank you for pointing this out!

**Pull request merged**